scrapers/PlayboyTV.yml

```diff
@@ -0,0 +1,29 @@
+name: "PlayboyTV"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - www.playboy.tv/watch/
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //h2[@class="title"]/text()
+      Date:
+        selector: //small[@class="subtitle"]/text()
+        replace:
+          - regex: .+(?:• Added on )(.+)
+            with: $1
+        # Date is not always present, but when it is, it is in a set format in
+        # the description field
+        parseDate: 02 January 2006
+      Details:
+        selector: //small[@class="subtitle"]/text()
+        replace:
+          - regex: (.+)(?:• Added on )(?:.+)
+            with: $1
+        # The field sometimes, has a description, sometimes has episode details.
+        # When it has episode details, the date can be stripped out.
+      Image: //div[@id="video"]/@poster
```

Comment:
shouldn't it be `//video[@id="video"]/@poster` ?

Reply:
No the id=video one is a div:
`//div[@id="video"]/@poster`

For video there is also this one:
`//video[@id="video_html5_api"]/@poster`

But neither seem to work cause the images are loaded via Javascript